tests/cupy_tests/sparse_tests/test_coo.py

```diff
@@ -0,0 +1,234 @@
+import unittest
+
+import numpy
+try:
+    import scipy.sparse
+    scipy_available = True
+except ImportError:
+    scipy_available = False
+
+import cupy
+import cupy.sparse
+from cupy import testing
+
+
+def _make(xp, sp, dtype):
+    data = xp.array([0, 1, 2, 3], dtype)
+    row = xp.array([0, 0, 1, 2], 'i')
+    col = xp.array([0, 1, 3, 2], 'i')
+    # 0, 1, 0, 0
+    # 0, 0, 0, 2
+    # 0, 0, 3, 0
+    return sp.coo_matrix((data, (row, col)), shape=(3, 4))
+
+
+def _make_unordered(xp, sp, dtype):
+    data = xp.array([1, 4, 3, 2], dtype)
+    row = xp.array([0, 2, 1, 0], 'i')
+    col = xp.array([0, 2, 3, 1], 'i')
+    # 1, 2, 0, 0
+    # 0, 0, 0, 3
+    # 0, 0, 4, 0
+    return sp.coo_matrix((data, (row, col)), shape=(3, 4))
+
+
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float32, numpy.float64],
+}))
+class TestCooMatrix(unittest.TestCase):
```

Comment:
Would you add tests for `dtype`, `data`, `row` and `col` attributes, including how `copy` parameter works?